### ISSUE TITLE
Missing information for the image

### ISSUE THREAD

**Issue by anna-guo-level-access** [anna-guo-level-access/a#24]
**Link to finding**  
https://anna-enterprise-org.hub.staging.dev.levelaccess.io/manual-evaluations/6a8f326b4b88ba4e9987bd49/results/800/view?skipScanStandards=true&status=toReview,open,fixed,partiallyFixed,notFixed,notReproducible,cannotBeFixed,validationRequested&linkedPropertyData=68d1a452844bdb0c39ead28e%7C68d1a452844bdb0c39ead281

**Severity**  
High

**URL**  
https://www.uab.edu/president/leadership

**Actual result**  
The content displayed in the image is not being read by the screen reader.

Important and relevant content is unavailable for screen reader users.

**Steps to reproduce**  
1. Go to the URL.
2. Navigate the page using the screen reader until reaching <the image>.

**Issue frequency**  
Individual

**Screen**  
Product

**Screenshot**  
https://download.essentialaccessibility.com/images/demo/issue%2011.png

**WCAG Success criteria**  
1.1.1

**WCAG Conformance level**  
A

**Effort**  
Low

**Instances**  
None provided

**Disabilities Affected**  
Visual, Auditory

**Recommendation**  
Ensure the <img> has an "alt" attribute with enough and relevant information.

Best practices for alternative text:
- Less than 125 characters long.
- Don't start the alt text with expressions such as "picture of..." or "image of..." 
- Describe the image with as much detail as possible.

Refer to:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img
https://www.w3schools.com/TagS/att_img_longdesc.asp
https://moz.com/learn/seo/alt-text

**Task in the Level Access Platform**  
https://anna-enterprise-org.hub.staging.dev.levelaccess.io/my-tasks/6a8f47a49202e91466ac9d37/6a8f48134b88ba4e9988056a

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Ivana Djeric from Level Access Platform.

Posted by Ivana Djeric via Level Access
